### PR TITLE
additional changes in the file

### PR DESCRIPTION
**Pull request by IgorPushnov**

### CODE REVIEW

**Review by shnax0210 — APPROVED** [review]

File: main.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const readWorld = require('./main');
+const calculateWordsStatistic = require('./main');
+
+test('string in the number of words', () => {
+	expect(calculateWordsStatistic(readWorld('testData')).toEqual
```
Comment (shnax0210) [inline main.test.js]: No need to import `./main` here, just pass test string to this function like:

```
test('check simple text statistic', () => {
  const testText = "Some test text, lala, test";
  expect(calculateWordsStatistic(testText)).toEqual({'some': 1, 'test': 1, 'text': 2, 'lala': 1})
}
```

File: main.js [hunk 2]
```diff
@@ -1,34 +1,7 @@
...
-  return collectStatistic(parseWords(objectForAnalysis.toString()));
-}
-readFiles(pathFiles);
-// console.log(textExample1);
-// console.log(textExample2);
-// console.log(textExample3);
-// console.log(textsForAnalysis);
-console.log(calculateWordsStatistic(textsForAnalysis));
+module.exports = readWorld;
+module.exports = calculateWordsStatistic;
```
Comment (shnax0210) [inline main.js]: No need to export anything from the file. This is our main file and all modules should be included here and used. So just have here:

```
const readFiles = require('./readfiles.js');
const calculateWordsStatistic = require('./analysis.js');

console.log(calculateWordsStatistic(readFiles('testData')));
```

File: main.test.js (hunk 1 above)
Comment (IgorPushnov) [reply main.test.js]: Changes are made

File: main.js (hunk 2 above)
Comment (IgorPushnov) [reply main.js]: Changes are made

File: main.js [hunk 3]
```diff
@@ -1,34 +1,5 @@
-const fs = require('fs');
-const path = require('path');
+const readWorld = require('./readfiles.js');
```
Comment (shnax0210) [inline main.js:1]: Why it's named `readWorld`?

**Review by shnax0210 — APPROVED** [review]

**Pull request merged**